distribution/lib/Standard/Test/0.0.0-dev/src/Problems.enso

```diff
@@ -1,8 +1,6 @@
 from Standard.Base import all
-import Standard.Test
 
-import Standard.Base.Error.Problem_Behavior
```

Comment:
What criteria were used to remove this?

This module is referenced in the type signature, although indeed removing the import will not break the file currently.

However, indeed I think we have a bit of a mess because of not-resolving type signatures - in some files the relevant types from signatures are imported and in others they are not (and I know I am responsible for this inconsitency in a few cases too - because we never set a rule of whether we should include such imports or not).

Shall we decide that for now we remove all imports which are not essential to compiling the given module? I guess this is more or less a robust rule.

I'd prefer to include the types referenced in signatures too, because this will be more future-compatible. However, since these are not verified currently, we will keep forgetting about some anyway. My line of reasoning was that its better to have only some types missing than lots of them, but I don't have a strong opinion here.

But I think we should develop a guideline regarding the imports which we can at least try to keep.

Reply:
`Problem_Behavior` is in `from Standard.Base import all` now as we use it everywhere.

Reply:
Oh, I did not notice that, sorry